### PR TITLE
Ensure manageiq packages are up to date

### PR DESCRIPTION
**Pull request by bdunne**
For some reason we're getting an up to date manageiq-appliance RPM but other manageiq-* RPMs are at older versions.  Force them to be updated to the latest available version.

### CODE REVIEW

File: kickstarts/base.ks.erb [hunk 1]
```diff
@@ -117,6 +117,9 @@ echo "" >> /etc/ssh/sshd_config
 # Remove packages not needed
 dnf -C -y --noplugins remove linux-firmware doxygen
 
+# Ensure packages are up to date
+dnf -y update manageiq-*
```
Comment (Fryguy) [inline kickstarts/base.ks.erb:121]: This won't work with productized builds, I don't think.

Comment (bdunne) [reply kickstarts/base.ks.erb]: Fixed

**Comment by bdunne** [comment]
I hit this again today, some packages are from March 30:
```
[root@localhost ~]# rpm -qa |grep manageiq
manageiq-ui-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-appliance-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-system-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-gemset-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-core-services-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-gemset-services-13.0.0-0.1.20210330000047.el8.x86_64
manageiq-appliance-tools-13.0.0-0.1.20210708000051.el8.x86_64
manageiq-release-13.0-1.el8.noarch
manageiq-core-13.0.0-0.1.20210330000047.el8.x86_64
```

**Comment by bdunne** [comment]
Followup: https://github.com/ManageIQ/manageiq-rpm_build/pull/166

**Pull request merged**